src/MyPlayer.cpp

```diff
@@ -17,7 +17,7 @@ class MyPlayer : public PlayerScript
     {
         if (sConfigMgr->GetOption<bool>("MyModule.Enable", false))
         {
-            ChatHandler(player->GetSession()).SendSysMessage("Hello World from Skeleton-Module!");
+            ChatHandler(player->GetSession()).PSendSysMessage(35410);
```

Comment:
I would like to put a better example here replacing the magic number 35410 with an enum and put the enum here, example:
```C++
enum AcoreString {
  HELLO_WORLD = 45410
};
....
ChatHandler(player->GetSession()).PSendSysMessage(HELLO_WORLD);
```

Reply:
Ready